spec/models/prison_spec.rb

```diff
@@ -144,8 +144,9 @@
       allocation_history.map do |ah|
         nomis_offender_id = ah.nomis_offender_id
         instance_double MpcOffender, "mpc_offender-#{nomis_offender_id}", offender_no: nomis_offender_id,
+                                                                          nomis_offender_id:,
                                                                           inside_omic_policy?: true,
-                                                                          probation_record: double,
+                                                                          case_information: double,
                                                                           released?: false
```

Comment:
Allowing the use of raw Offender fields rather than MPCOffender to allow passing in of Offender/MPCOffender into methods above.